cloudlift/deployment/changesets.py

```diff
@@ -59,7 +58,6 @@ def create_change_set(client, service_template_body, stack_name,
             ChangeSetName=create_change_set_res['Id']
         )
         log_bold("Done. Bye!")
-        exit(0)
```

Comment:
Even without `exit(0)`, the command will exit. So removing it.

Reply:
Is this also confirmed to exit with 0?

Reply:
Good catch. While I tested, I found one more problem.

Actually the change-set creation had a bug. If there's no change it logs `Changeset creation failed!` and returns `None` to `ServiceCreater.update`. But the ServiceCreator doesn't check and fails trying process. 


I've fixed and tested both scenarios.


<img width="1020" alt="Screen Shot 2020-04-22 at 7 43 25 PM" src="https://user-images.githubusercontent.com/309563/79993103-029c0780-84d2-11ea-8b6c-a03afeb2fabb.png">
<img width="921" alt="Screen Shot 2020-04-22 at 7 43 17 PM" src="https://user-images.githubusercontent.com/309563/79993115-06c82500-84d2-11ea-82c4-f0fb1538c94e.png">

Reply:
It always annoyed me that _No changes_ resulted in changeset creation failure instead of a warning or info that changeset creation was skipped.